templates/post.html

```diff
@@ -65,6 +65,21 @@ <h1>{{ page.title }}</h1>
     <div class="post-main">
         <div class="post-content">
             {{ page.content | safe | markdown | safe }}
+            <div class="callout">
+                <h2>The Foundation needs you</h2>
+                <p>
+                    The Matrix.org Foundation is a non-profit and only relies on donations to operate. Its core mission
+                    is to maintain the Matrix Specification, but it does much more than that.
```

Comment:
Might not be something to solve now but it might be nice to link to some place on the "more than that" part which extends on that.

Reply:
This is likely something we will work on when launching the fundraising campaigns!